components/raftstore/src/store/msg.rs

```diff
@@ -229,6 +239,11 @@ impl<S: Snapshot> ReadCallback for Callback<S> {
     fn set_result(self, result: Self::Response) {
         self.invoke_read(result);
     }
+
+    fn tracker(&self) -> Option<&TrackerToken> {
```

Comment:
`WriteCallback` has `trackers` and now `ReadCallback` has `tracker`. I think it will confuse people when calling these functions (because we don't specify the trait when calling them).

One option is to unify them all to `trackers` and `trackers_mut`, and check if it's really a read index before observing the duration.

Another option is to change the method name to `write_trackers`  and `read_tracker` for clarification.

Reply:
Good idea ! I perfer the second one~